### DISCUSSION TITLE
[newbie] Persistence on a Raspberry Pi?

### DISCUSSION THREAD

**Discussion by akavel** [beakerbrowser/beaker#1683] [General] [answered]
Hi! I have a Raspberry Pi where I'd like to "persist" any content I created in the Beaker Browser: should I use https://github.com/beakerbrowser/homebase for that?

Is there some way I can easily make the RPi automatically "mirror" anything ***I*** have created in Beaker?

Is there some way I can easily make the RPi mirror also all the stuff created by others, that I marked as wanting to mirror in Beaker?

**Comment by pfrazee** [comment, marked as answer]
Homebase is currently out of date. The current answer is to use `hyperdrive-daemon`, there is a "mirror" command in development (see https://github.com/hypercore-protocol/hyperdrive-daemon/issues/49#issuecomment-629726837). There may be other options in the community as well.

**Reply by pfrazee** [reply]
> Is there some way I can easily make the RPi mirror also all the stuff created by others, that I marked as wanting to mirror in Beaker?

That's something we're considering as well
